x-pack/plugins/apm/server/lib/rum_client/get_page_load_distribution.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getRumOverviewProjection } from '../../../common/projections/rum_overview';
+import { mergeProjection } from '../../../common/projections/util/merge_projection';
+import {
+  Setup,
+  SetupTimeRange,
+  SetupUIFilters,
+} from '../helpers/setup_request';
+
+export async function getPageLoadDistribution({
+  setup,
+  minPercentile,
+  maxPercentile,
+}: {
+  setup: Setup & SetupTimeRange & SetupUIFilters;
+  minPercentile?: string;
+  maxPercentile?: string;
+}) {
+  const projection = getRumOverviewProjection({
+    setup,
+  });
+
+  const params = mergeProjection(projection, {
+    body: {
+      size: 0,
+      query: {
+        bool: projection.body.query.bool,
+      },
+      aggs: {
+        durationMinMax: {
+          min: {
+            field: 'transaction.duration.us',
+            missing: 0,
+          },
+        },
+        durationPercentiles: {
+          percentiles: {
+            field: 'transaction.duration.us',
+            percents: [50, 80, 90, 95, 99],
```

Comment:
80 isn't commonly used, Isn't it supposed to be `75`?

Reply:
will be making it 75.